CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
should be scheduled as a new release

Reply:
yes that's why it is documented under unreleased... it's a cosmetic change only... so no immediate release needed ;)